### PR TITLE
Server init

### PR DESCRIPTION
**Pull request by thomasjuin1**
* **Please check if the PR fulfills these requirements**
- [ X ] The commit message follows our guidelines
- [ X ] Tests for the changes have been added (for bug fixes/features)
- [ X ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Features

* **What is the current behavior?** (You can also link to an open issue here)
Nothing

* **What is the new behavior (if this is a feature change)?**
The new behavior is a Server with Select, write & read FDS. 
Parsing for all flags.
It have the team creation, the clients inits, and commands handling.

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
No breaking changes

* **Other information**:
Nothing

### CODE REVIEW

**Review by Sinan-Karakaya — APPROVED** [review]
Alloc are checked: ✔️ 
Norm seems good: ✔️ 
Code is documented: ✔️ 
Looks clean enough: ✔️ 
Actions are good: ✔️

**Review by RoheeAxel — APPROVED** [review]
It seems good enough

**Review by noerio — APPROVED** [review]
Code clean, Well documented

File: zappy_server/src/parsing/fill_parsing.c [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    }
+    return 0;
+}
+
+int fill_parsing(char **args, parsing_t *parsing)
+{
+    if (!parsing || !args)
+        return 84;
+    for (size_t i = 1; args[i]; i++) {
+        if (strcmp(args[i], "-p") == 0)
```
Comment (JasonLAGOUTE) [inline zappy_server/src/parsing/fill_parsing.c:57]: should try getopt

**Review by JasonLAGOUTE — COMMENTED** [review]
Good, change malloc to calloc and return int with ssize_t if it's possible.

**Review by noerio — COMMENTED** [review]
Code seems good, not all files are documented

**Pull request merged**